src/pages/billing/Project.js

```diff
@@ -73,6 +79,7 @@ const WorkspaceCard = Utils.memoWithName('WorkspaceCard', ({ workspace, isExpand
   return div({ role: 'listitem', style: { ...Style.cardList.longCardShadowless, flexDirection: 'column' } }, [
     h(IdContainer, [id => h(Fragment, [
       div({ style: workspaceCardStyles.row }, [
+        statusIcon,
```

Comment:
With the other changes in place try:
```suggestion
statusIcon && icon(statusIcon, { size: 16 }),
```